azure-ad.sgnl.yaml

```
@@ -581,6 +625,24 @@ relationships:
     toAttribute: User.id
     # [TODO sc-11020] Give these a unique ID, rename both of these to Member,
     # and keep corresponding path relationships unique below
+  RoleMember:
+    name: MemberOf
+    displayName: Role To Role Member
+    fromAttribute: Role.id
+    toAttribute: RoleMember.roleId
```

Please make sure to keep the name, displayName, fromAttribute and toAttribute consistent with the GroupMember counterparts.

For example, 
a `GroupMember` relationship equivalent for a role would look like
```suggestion
  RoleMember:
    name: Member
    displayName: RoleMember
    fromAttribute: RoleMember.memberId
    toAttribute: Role.roleId
```


Please make similar changes to ALL relationships listed in https://github.com/SGNL-ai/catalog-sgnl-templates/pull/49/files#r1541841918